lib/util.js

```diff
@@ -371,10 +371,11 @@ function formatValue(ctx, value, recurseTimes) {
   // Look up the keys of the object.
   var keys = Object.keys(value);
   var visibleKeys = arrayToHash(keys);
+  var symbolKeys = Object.getOwnPropertySymbols(value);
```

Comment:
Well… it seems a bit weird to list only enumerable string properties but all Symbol properties by default?